### DISCUSSION TITLE
Unrelated

### DISCUSSION THREAD

**Discussion by MidnightKittenCat** [absadiki/subsai#103] [Ideas]
Would it be possible to make another project for live translation? (Translating audio to text), the technology is definitely there to be able to do it and as far as I’ve seen I can’t find any decent ones that do the job well, would you be able to look into this at some point? I’m sure I’m not the only one.

could be used for over seas zoom meetings etc (with gpu support hopefully)

**Comment by absadiki** [comment]
I had a similar idea a while ago but couldn't find time to do it. 
But, AFAIK, zoom already supports this out of the box, so if it's just for over seas zoom meetings you won't need any other software to have live translation.

**Reply by MidnightKittenCat** [reply]
It's outside of zoom unfortunately.

**Reply by MidnightKittenCat** [reply]
https://github.com/MotazSabri/Hanami-release

This is the closest I could find to what im looking for and it does the job, except its very slow and takes around 15s (I'm not even sure if its using my gpu) 

It also requires internet connection which is another downside.

**Reply by absadiki** [reply]
> It's outside of zoom unfortunately. 

What do you mean ? 

> [MotazSabri/Hanami-release](https://github.com/MotazSabri/Hanami-release)
> 
> This is the closest I could find to what im looking for and it does the job, except its very slow and takes around 15s (I'm not even sure if its using my gpu)
> 
> It also requires internet connection which is another downside.

This is a closed source software so I cannot tell, but yes from their description I can see that they are using third party APIs, so no GPU is needed.

**Reply by MidnightKittenCat** [reply]
![HanamiGUI_8nr4vvd4jN](https://github.com/abdeladim-s/subsai/assets/66114416/dda5ed2d-2d5b-4276-a445-299d80efd93c)

Sort of how Hanami does it, I'm looking for an on top program that translates my system audio from (Example: Japanese) to English using gpu support and locally asw.
